lua/ulx/modules/sh/cfc_trainfuck.lua

```diff
@@ -0,0 +1,42 @@
+CATEGORY_NAME = "Fun"
+local trainSounds = { "ambient/alarms/razortrain_horn1.wav",
+                      "ambient/machines/usetoilet_flush1.wav",
+                      "ambient/machines/wall_crash1.wav",
+                      "garrysmod/balloon_pop_cute.wav",
+                      "garrysmod/save_load1.wav"
+                    }
+                    
+local function trainFuck( callingPlayer, targetPlayers )
+    for _, ply in pairs( targetPlayers ) do
+    
+        local soundPlay = trainSounds[ math.random( 1, 5 ) ]
```

Comment:
```suggestion
        local soundPlay = trainSounds[math.random( 1, 5 )]
```
this newline aint needed, and we don't typically do spaces around square brackets, else you end up with code like
```lua
myArray[ 5 ]
```
for consistency, and that looks a little odd